sp800-63-3/sec6_xAL.md

```diff
@@ -173,17 +173,44 @@ The AAL selection does not mean the digital service provider will need to issue
 
-FAL2 and higher require the assertion itself be encrypted such that the intended RP is the only party that can decrypt it. This method improves the enforcement of audience restriction at RPs (since an unintended RP won't be able to decrypt an assertion) and increases privacy protection by protecting the assertion message itself beyond having it be passed along authenticated protected channels. RPs that allow front-channel assertion presentation should require at least FAL2 to protect the content of the assertion, since the assertion can be seen by the subscriber and handled by the subscriber's browser.
+**Figure 6-3 Selecting FAL**
+</div>
 
-FAL3 further requires the subscriber prove possession of a key in addition to the ability to present an assertion or assertion reference. This allows the RP to strongly verify the binding of the assertion to the subscriber by means of a subscriber-held key. This key is referenced in the assertion and represents the subscriber.
+<div class="text-left" markdown="1">
+<table style="width:100%">
+  <tr>
+    <td><img src="sp800-63-3/media/fal-step1.png" alt="FAL Step 1"/></td>
+  </tr>
+  <tr>
+   <td>Step 1 asks agencies to look at the potential impacts of a federation failure. In other words, what would occur if an unauthorized user could compromise an assertion. Compromise could include being able to replay an assertion to impersonate a valid user or an assertion leaking information through the browser. Risk should be considered from the perspective of the organization and to a valid user, since one may not be negatively impacted while the other could be significantly harmed. Agency risk management processes should commence with this step.</td> 
+  </tr>
+  <tr>
+    <td><img src="sp800-63-3/media/fal-step2.png" alt="FAL Step 2"/></td>
+  </tr>
+  <tr>
 
-Increasing the FAL increases the complexity of the deployment and management of a federation system, as RP keys need to be managed at FAL2 and FAL3, and subscriber keys need to be managed at FAL3. Therefore, RPs should add advanced functionality where it is feasible and warranted for the application.
+   <td>FAL2 is required when any personal information is passed in an assertion. Since the other paths in this decision tree already drive the agency to FAL2 or higher, the question of personal information is only raised at this point. That said, personal information release at all FALs should be considered when performing the risk assessment. FAL2 or higher is required when any personal information is contained in an assertion, as the audience and encryption requirements do not adequately protect personal information from being released. Release of even self-asserted personal information requires assertion protection via FAL2. Even though self-asserted information can be falsified, most users will provide accurate information to benefit from the digital service.</td> 
```

Comment:
I'd remove the second sentence:

> Since the other paths ... raised at this point.

And change the third sentence to:

> Personal information release ...

Add this note to cover things like OIDC's UserInfo Endpoint:

> When personal information is available to the RP via an authorized API call, such information need not be included in the assertion itself. Since the assertion no longer includes personal information, it need not be protected at FAL2.